### PR TITLE
Added Silencing overview guide.

### PR DESCRIPTION
**Pull request by mercul3s**
Ported from sensu-alpha docs.

### CODE REVIEW

**Review by nikkictl — APPROVED** [review]
💛

File: content/sensu-core/2.0/overview/silencing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Intro to Silencing"
+date: 2018-02-22T10:49:25-08:00
+description:
```
Comment (nikkiattea) [inline content/sensu-core/2.0/overview/silencing.md]: I am curious why we have a date and no description though.

**Review by nikkictl — COMMENTED** [review]
Just a question..

File: content/sensu-core/2.0/overview/silencing.md (hunk 1 above)
Comment (mercul3s) [reply content/sensu-core/2.0/overview/silencing.md]: Oh, probably because I generated it and forgot to edit :)

**Pull request merged**